src/__tests__/Search.test.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
```

Comment:
This is not the latest version of the output your code produces, you should run the tests again and commit the latest snapshots before submitting for review